### DISCUSSION TITLE
希望可以增加快捷键翻页的支持

### DISCUSSION THREAD

**Discussion by HofNature** [xiaoyaocz/flutter_dmzj#134] [Ideas]
在一些电子墨水阅读器或者平板电脑上，通常具有实体翻页键(经常是使用键盘上PageUP和PagesDown键的键码)。希望能够支持在小说列表和阅读页面使用实体翻页键翻页的功能。
如果有可能也希望支持音量键翻页。

**Comment by HofNature** [comment]
我个人在使用博阅T62D(安卓电子书)，它的实体翻页键对应的就是PageUP和PageDown。我手头的小米平板5附带的笔上的两个按键也是这两个。

**Comment by xiaoyaocz** [comment]
#140
